README.md

```diff
@@ -44,3 +44,39 @@ yarn run stylelint
 ```
 npm run stylelint
 ```
+
+## Prettier
+
+This config also includes a prettier config which can be extended to format `.scss`.
+
+Install [`prettier-stylelint`](https://github.com/hugomrdias/prettier-stylelint):
+
+```
+$ yarn add --dev prettier-stylelint
```

Comment:
I like to omit `$` to make it easier to copy-paste the whole line (e.g. using triple-click instead of having to select the important part), your call.